thriftserver/server/src/test/scala/org/apache/livy/thriftserver/ThriftServerSuites.scala

```diff
@@ -134,6 +134,29 @@ class BinaryThriftServerSuite extends ThriftServerBaseTest with CommonThriftTest
       statement.close()
     }
   }
+
+  test("invalid sql") {
```

Comment:
please add the JIRA number and provide a better description
```suggestion
  test("LIVY-571: Invalid query returns a meaningful exception") {
```